docker/build.sh

```diff
@@ -175,8 +175,9 @@ function docker_cmd()
 
         docker build -t "${DEFAULT_IMAGE}:${DOCKER_TAG}" .
 
-        echo "scanning docker image..."
-        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "${DEFAULT_IMAGE}":"${DOCKER_TAG}"
+# TODO: enable trivy again https://broadworkbench.atlassian.net/browse/IA-2291
+#        echo "scanning docker image..."
+#        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "${DEFAULT_IMAGE}":"${DOCKER_TAG}"
```

Comment:
Maybe we could still run it without `--exit-code 1` to report the errors but not fail the build?

Reply:
Also did it in my PR: https://github.com/DataBiosphere/leonardo/pull/1652

Reply:
oh I see..

> Maybe we could still run it without --exit-code 1 to report the errors but not fail the build?
Good idea!

I'll wait for your PR then if you've already made the change :)